README.md

```diff
@@ -20,33 +161,52 @@ pnpm install
 pnpm start
 ```
 
-### 📦 COMING SOON - TypeScript Packages
+## 🎓 For students and academics
+
+Deepnote is **free for students and educators**! Get unlimited access to all core features, cloud compute, and real-time collaboration for your research and teaching.
 
-Reusable packages and libraries powering Deepnote’s notebook, runtime, and collaboration features.
+**Learn more:** [deepnote.com/education](https://deepnote.com/education?utm_source=github&utm_medium=github&utm_campaign=github&utm_content=readme_main)
+
+---
+
+## 🙌 Need help?
+
+- 👥 Join our [Community]()!
+- 💬 [Open an issue](https://github.com/deepnote/deepnote/issues/new) for bug reports or feature requests
```

Comment:
Yes but we want issues on the right repos. So while the comments above I've made are about linking to the published artifacts we also need links to the source code repos and a note that issue should be opened in the repo for the product the issue relates to.